config/networking.yml

```diff
@@ -36,23 +36,27 @@ metadata:
     #@overlay/match missing_ok=True
     cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
 
-#@ cf_namespaces = [data.values.system_namespace, data.values.workloads_namespace , "cf-db", "cf-blobstore", "metacontroller"]
+#@ kind_deployment = overlay.subset({"kind":"Deployment"})
+#@ kind_replica_set = overlay.subset({"kind":"ReplicaSet"})
+#@ kind_stateful_set = overlay.subset({"kind":"StatefulSet"})
+#@ kind_daemon_set = overlay.subset({"kind":"DaemonSet"})
+#@ kind_job = overlay.subset({"kind":"Job"})
+#@ is_pod = overlay.or_op(kind_deployment, kind_replica_set, kind_stateful_set, kind_daemon_set, kind_job)
```

Comment:
The nit-est of picks. We're probably never going to directly deploy a pod, but kind_pod is technically absent. It might be more clear to call this contains_pod or something like that? I'm not going block the PR on this, so feel free to merge.